### PR TITLE
app-containers/crun: bump to 1.11.2

### PR DESCRIPTION
**Pull request by robertgzr**
Signed-off-by: Robert Günzler <r@gnzler.io>

### CODE REVIEW

**Comment by rahilarious** [comment]
Upstream latest is 1.9.2. Lets try to get this merged once you bump to latest

File: app-containers/crun/crun-1.9.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
```
Comment (juippis) [inline app-containers/crun/crun-1.9.1.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{10..12} )
```
IF 3.12 works. 3.9 is disabled already in the repo.

**Review by juippis — COMMENTED** [review]
Yeah, could you update this to 1.9.2 and let's get that merged?

**Comment by rahilarious** [comment]
@robertgzr v1.10 released. This package is crucial for Containers stack. Please take care of this

**Comment by rahilarious** [comment]
@robertgzr 1.11 released. Ping? Should I open bug on b.g.o?

**Comment by robertgzr** [comment]
@juippis @rahilarious sorry for the delay :/

File: app-containers/crun/crun-1.9.1.ebuild (hunk 1 above)
Comment (robertgzr) [reply app-containers/crun/crun-1.9.1.ebuild]: does the fixup commit work for you?

**Comment by rahilarious** [comment]
@robertgzr Could you squash both commits into one?

**Comment by Flowdalic** [comment]
I think you may want to keep src_test() even though you add RSTRICT=test. Also,  what about: ":# Can drop autotools/eautoreconf after next release & glibc patch gone".  I don't see a patch in the ebuild, so maybe it got upstreamed and we can also drop autotools/eautoreconf?

**Comment by robertgzr** [comment]
> I think you may want to keep src_test() even though you add RSTRICT=test. 

why is that?

> Also, what about: ":# Can drop autotools/eautoreconf after next release & glibc patch gone". I don't see a patch in the ebuild, so maybe it got upstreamed and we can also drop autotools/eautoreconf?

+1

**Review by juippis — COMMENTED** [review]
```
libtool: link: x86_64-pc-linux-gnu-gcc -I /var/tmp/portage/app-containers/crun-1.11.2/work/crun-1.11.2/libocispec/src -I /var/ tmp/portage/app-containers/crun-1.11.2/work/crun-1.11.2/libocispec/src -D CRUN_LIBDIR=\"/usr/lib64/crun\" -march=native -O2 -  pipe -frecord-gcc-switches -fPIC -Wl,-O1 -Wl,--as-needed -Wl,--defsym=__gentoo_check_ldflags__=0 -fPIC -o .libs/crun src/crun- crun.o src/crun-run.o src/crun-delete.o src/crun-kill.o src/crun-pause.o src/crun-unpause.o src/crun-oci_features.o src/crun-  spec.o src/crun-exec.o src/crun-list.o src/crun-create.o src/crun-start.o src/crun-state.o src/crun-update.o src/crun-ps.o src/crun-checkpoint.o src/crun-restore.o src/libcrun/crun-cloned_binary.o  ./.libs/libcrun.so -lseccomp -lgcrypt -lyajl
/usr/lib/gcc/x86_64-pc-linux-gnu/13/../../../../x86_64-pc-linux-gnu/bin/ld: ./.libs/libcrun.so: undefined reference to         `cap_to_name'
collect2: error: ld returned 1 exit status
make: *** [Makefile:1505: crun] Error 1
 * ERROR: app-containers/crun-1.11.2::gentoo failed (compile phase):
 *   emake failed
```
```[ERROR] >>> Not all runs were successful.
atom: =app-containers/crun-1.11.2, USE flags: '-bpf -caps -criu seccomp -static-libs'
atom: =app-containers/crun-1.11.2, USE flags: '-bpf -caps criu -seccomp -static-libs'
```
If I had to guess, `-caps` is to blame. Could you try and verify this?

**Comment by robertgzr** [comment]
hmm looks like they introduced dependency on libcaps that isn't guarded by ifdefs, looks easy to fix, i'll ask upstream

**Comment by robertgzr** [comment]
fix merged https://github.com/containers/crun/pull/1345

**Review by juippis — APPROVED** [review]
Awesome, thank you!

**Pull request closed without merging**